spec/svg_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'SVG files' do
-  Dir[Rails.root.join('**', '*.svg')].each do |svg_path|
+  Dir[Rails.root.join('**', '*.svg')].reject { |f| f['node_modules'] }.each do |svg_path|
```

Comment:
is this rejecting files that contain `node_modules` in the path? I would be more explicit than using the indexing syntax"

```
.reject { |f| f.include?('node_modules') }
```